docs/best_practices_guide.adoc

```diff
@@ -507,21 +507,24 @@ This is almost always more desirable than an empty list, string, etc.
 This is done primarily because if you're registering the result of the
 installation and you have two conditional tasks based on whether or not yum or
 dnf are in use you'll end up inadvertently overwriting the value. It also
-reduces duplication. name= and state=present are common between dnf and yum
-modules.
```

Comment:
Should lines 500-509 be removed as well, since syncing parameters between yum/dnf modules is no longer needed?